package.json

```diff
@@ -59,12 +58,11 @@
     "classificator": "^0.3.4",
     "clsx": "^1.1.1",
     "cozy-bar": "7.12.3",
-    "cozy-client": "24.10.1",
+    "cozy-client": "35.3.1",
```

Comment:
Why did you need to update cozy-client?

Reply:
I was still stuck and copied coachC02 to be safe. Should I downgrade it?

Reply:
On the contrary, it's good to upgrade, but I was wondering the reason